### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Nemo1710** [martin-ger/esp_wifi_repeater#93]
Really it's a really good job, but I have a question, it's possible to use GPIO inputs with MQTT at the same time the router functions

**Comment by martin-ger** [comment]
Yes, it is. If you uncomment USER_GPIO_IN  and USER_GPIO_OUT at the end of user_config.h and recompile it, you can use one input and one output pin via MQTT.

**Comment by Nemo1710** [comment]
This is Great, it would be really that i could do that with the ADC too
